pages/docs/database-connections/databricks.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Databricks is a flexible data analytics platform with a wide variety of built-in services
+---
+
+# Databricks
+
+[Databricks](https://www.databricks.com) is a data analytics platform built on top of Apache Spark. 
+
+Databricks allows you to combine the robustness of a data warehouse with the flexibility of a data lake via their 
+[Lakehouse architecture](https://www.databricks.com/product/data-lakehouse).
+It provides a unified platform that offers a wide variety of services, allowing you to store and process large amounts of data quickly and efficiently, 
+including real-time analytics. 
+
+Databricks can be especially useful for machine learning/artificial intelligence, data engineering, and data science applications.
+
+### How to get set up
+
+1. Sign up for a Databricks account or trial and create a workspace. You can find instructions on how to set up an account and workspace 
+[here](https://docs.databricks.com/en/getting-started/index.html). 
+2. Databricks automatically configures a "Starter Warehouse" for new users. 
+You can find it by clicking the "SQL Warehouses" link in the left sidebar of your Databricks workspace. 
+You can find instructions on how to create or configure a warehouse [here](https://docs.databricks.com/en/compute/sql-warehouse/create-sql-warehouse.html).
+3. Generate a [personal access token](https://docs.databricks.com/en/dev-tools/auth/pat.html#pat-user) in Databricks for a user with access to the data you want to query:
+    * In the top right corner of your Databricks workspace, click the user icon and navigate to "User Settings".
+    * In the left pane, click into the "Developer" controls. Click the "Manage" button in the "Access tokens" section.
+    * Click "Generate New Token". You will be asked to configure the token's lifespan. Finally, click "Generate".
+    * Copy the token and save it somewhere safe. You will not be able to access it again.
+4. In the sidebar of your workspace, click on "SQL Warehouses" under the "SQL" section. 
+You will see a list of your warehouses. Click on the name of the warehouse you want to connect to.
+Then, navigating to the "Connection details" tab, you can find the hostname and HTTP path of your warehouse.
+5. In Hashboard, go to the [Data sources](https://hashboard.com/app/p/data-sources) page.
+6. Click `+ Add connection`, select `Databricks` and fill out the fields below.
```

Comment:
Would be valuable to normalize how we cite buttons as either `text` or "text" within this section. I think we do indeed usually go with `text`.